private-data-transfer-ui/src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
I would just leave this out and add it to your `.gitignore`, it gets autogenerated by CRA each time but is useless
https://stackoverflow.com/questions/53527763/create-react-app-typescript-auto-generating-react-app-env-d-ts-in-src-director